Widgets/Widgets/PictureOfTheDayWidget.swift

```diff
@@ -0,0 +1,248 @@
+import WidgetKit
+import SwiftUI
+import WMF
+import UIKit
+
+// MARK: - Widget
+
+struct PictureOfTheDayWidget: Widget {
+	private let kind: String = WidgetController.SupportedWidget.pictureOfTheDay.identifier
+
+	public var body: some WidgetConfiguration {
+		StaticConfiguration(kind: kind, provider: PictureOfTheDayProvider(), content: { entry in
+			PictureOfTheDayView(entry: entry)
+		})
+		.configurationDisplayName("Picture of the day")
```

Comment:
These should be translated, no?

Reply:
Indeed, localization is on my list of what to figure out how to do "the SwiftUI" way. If you figure out best practices here, lemme know, otherwise I'll dig into it again when I next pass this widget. As I understood it, the pictures of the day are English-only, so this wasn't an immediate priority I had when first working on this widget.

Reply:
Sounds good. In the widget I'm working on, I'm using translated strings just like anywhere else in the app. And since we have similar work for the explore feed, all the strings I needed already existed.